client/src/app/site/motions/modules/amendment-list/amendment-list.component.html

```diff
@@ -136,5 +136,15 @@ <h2>
             <mat-icon>clear</mat-icon>
             <span translate>Deselect all</span>
         </button>
+
+        <div *osPerms="'motions.can_manage'; or: 'motions.can_manage_metadata'">
+            <mat-divider></mat-divider>
+            <os-motion-menu-list [selectedMotions]="selectedRows" (action)="multiselectWrapper($event)">
+                <button mat-menu-item (click)="openExportDialog()">
```

Comment:
please remove and move into `os-motion-menu-list`.
Just like in the normal motion list, the list of amendments should also be able to export selected motions